hangulime.js

```diff
@@ -376,9 +376,9 @@ var jamo_med_u_ae   = new Jamo(jamo_med_u,   "no", JAMO_MEDIAL, 'ᆊ', undefined
 var jamo_med_u_ye   = new Jamo(jamo_med_u,   "nP", JAMO_MEDIAL, 'ᆌ', undefined);
 var jamo_med_u_u    = new Jamo(jamo_med_u,   "nn", JAMO_MEDIAL, 'ᆍ', undefined);
 var jamo_med_yu_a   = new Jamo(jamo_med_yu,  "bk", JAMO_MEDIAL, 'ᆎ', undefined);
-var jamo_med_yu_eo  = new Jamo(jamo_med_yu,  "bj", JAMO_MEDIAL, 'ᆏ', undefined);
+var jamo_med_yu_eo  = new Jamo(jamo_med_yu,  "bu", JAMO_MEDIAL, 'ᆏ', undefined);
 var jamo_med_yu_e   = new Jamo(jamo_med_yu,  "bp", JAMO_MEDIAL, 'ᆐ', undefined);
-var jamo_med_yu_eo  = new Jamo(jamo_med_yu,  "bj", JAMO_MEDIAL, 'ᆑ', undefined);
+var jamo_med_yu_eo  = new Jamo(jamo_med_yu,  "bu", JAMO_MEDIAL, 'ᆑ', undefined);
```

Comment:
issue: As above, here's what this line should be:

```suggestion
var jamo_med_yu_yeo  = new Jamo(jamo_med_yu,  "bu", JAMO_MEDIAL, 'ᆑ', undefined);
```